components/sup/src/manager/service/hooks.rs

```diff
@@ -144,7 +141,7 @@ pub trait Hook: fmt::Debug + Sized {
 
     fn path(&self) -> &Path;
 
-    fn template(&self) -> &Template;
+    fn registry(&self) -> &Registry;
```

Comment:
Yeah I was a 2/10 on this name but it was a hell of a lot better than Template which is what is actually being registered to this thing. I was also thinking `Renderer` so maybe `Template` was a bad name but `TemplateRenderer` is spot on. I'll make that change